addons/sys_gui/fnc_onInventoryRadioDoubleClick.sqf

```diff
@@ -19,4 +19,7 @@ TRACE_1("enter", _this);
 
 params ["_object","_container","_radioId"];
 
+// When a radio is opened set acre_player to the current player
+acre_player = acre_current_player;
```

Comment:
This is not the only way of opening radios, there are also keybind and ACE Self-Interaction. What is the goal here?

Reply:
When remote controlling a unit the interaction menu they will use their unit's radios instead of the remote-controlled unit's radio. This will switch to the remote-controlled unit's radios if they open a radio via the inventory.

Reply:
This will be improved when I do the PR for Zeus radios since right now it is unclear what radios Zeus will be using, especially if they are a virtual entity and not a regular unit.

Reply:
Ah I see, and that's not a problem when opening with keybind etc. Sounds good.